selenium_test.py

```diff
@@ -0,0 +1,222 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+import pandas as pd
+import time
+
+KEYWORDS_FILE = 'textfile.txt'
+RANKED_OUTPUT_FILE = 'Output.csv'
+START_YEAR = "1917"
+END_YEAR = "1971"
+
+
+class resultLists:
+    title_list = []
+    description_list = []
+    category_list = []
+    keyword_list = []
+    link_list = []
+
+lists = resultLists()
```

Comment:
Similarly, this is not better either. I'm okay with having `START_YEAR` and `END_YEAR` as global constants (although they really should be read in from the same file as the keywords), but there's no need for this global list. General good coding practice is to keep the scope of each variable as small as possible.